zipkin-server/src/main/java/zipkin/server/ZipkinQueryApiV1.java

```diff
@@ -43,6 +44,7 @@
  */
 @RestController
 @RequestMapping("/api/v1")
+@CrossOrigin("${zipkin.query.allowed-origins:'*'}")
```

Comment:
I don't know if these single quotes around the `*` are needed or make a difference.
A few UT/IT to verify the situations we are covering would, of course, be best. Manual tests in the meantime might suffice.